games/clappy_trio/clappy_trio.bs

```diff
@@ -522,7 +522,7 @@ script script_clappy_trio_practice
     rest 6
     init_tutorial scene_clappy_trio_skipped_practice
     print_text_f D_0805af14
-    clappy_trio_hide_textbox 1
+    clappy_trio_set_textbox_visibility 1

```

Comment:
this can probably be TRUE or FALSE now, and maybe the name could be made a bit shorter, like just `textbox_shown`